libs/api-bundle/tests/Security/ManageApiToken/ManageApiTokenTest.php

```diff
@@ -70,4 +69,33 @@ public function testHasScope(): void
         self::assertFalse($token->hasScope('other:scope'));
         self::assertFalse($token->isSuperAdmin());
     }
+
+    public function testHasFeature(): void
+    {
+        $token = ManageApiToken::fromVerifyResponse([
+            'id' => 99994,
+            'description' => 'Adam test',
+            'created' => '2024-03-21T12:26:43+0100',
+            'lastUsed' => '2024-03-21T12:26:54+0100',
+            'expires' => null,
+            'isSessionToken' => false,
+            'isExpired' => false,
+            'isDisabled' => false,
+            'scopes' => [],
+            'type' => 'super',
+            'user' => [
+                'id' => 3801,
+                'name' => 'Adam Výborný',
+                'email' => 'adam.vyborny@keboola.com',
```

Comment:
jako lepší by bylo mít tu john.doe@example.com když je to public

Reply:
Upravil jsem v celym testu